.github/workflows/ci.yml

```diff
@@ -1,9 +1,10 @@
 name: Lint and Test Charts
 on:
-  push:
+  pull_request:
     branches:
-      - 'main'
-  pull_request: {}
+      - dev-v2.6
```

Comment:
We could include some regex into the branches: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags